collections/_article/dev-snapshot-godot-4-6-beta-1.md

```diff
@@ -0,0 +1,380 @@
+
+And more:
+- Add MeshInstance3D primitive conversion options ([GH-101521](https://github.com/godotengine/godot/pull/101521)).
+- Add MultiMesh physics interpolation for 2D transforms (MultiMeshInstance2D) ([GH-107666](https://github.com/godotengine/godot/pull/107666)).
+
+### Platforms
+
+#### Android
+
+[Anish](https://github.com/syntaxerror247) took to the implementation of Storage Access Framework (SAF) support ([GH-112215](https://github.com/godotengine/godot/pull/112215)). Previously, accessing non-media files on external storage required the `MANAGE_EXTERNAL_STORAGE` permission, which is very broad and discouraged by Google. Now, projects can access files through the system's file picker instead, and this method doesn't require any permissions at all!
+
+Thanks to the efforts of [David Snopek](https://github.com/dsnopek) and [Logan Lang](https://github.com/devloglogan), Godot is now able to initiate Gradle builds on Android ([GH-111732](https://github.com/godotengine/godot/pull/111732)). By sending commands to a companion app that provides a full Linux-like build environment to run Gradle, Godot itself remains lightweight while keeping setup easy by including everything you need within the companion app.
+
+And more:
+- Editor: Add game speed control options in game menu bar ([GH-111296](https://github.com/godotengine/godot/pull/111296)).
+- Editor: Adjust script editor size for virtual keyboard ([GH-112766](https://github.com/godotengine/godot/pull/112766)).
+- Editor: Persist fullscreen setting ([GH-112246](https://github.com/godotengine/godot/pull/112246)).
+- Export: Add export option to use "scrcpy" to run project from editor ([GH-108737](https://github.com/godotengine/godot/pull/108737)).
+- Tests: Add Android instrumented tests to the `app` module ([GH-110829](https://github.com/godotengine/godot/pull/110829)).
+
+#### Linux
+
+[Deralmas](https://github.com/deralmas) has continued on their quest to make Wayland a first-class window protocol with the exciting introduction of game embedding ([GH-107435](https://github.com/godotengine/godot/pull/107435)). This was one of the final major roadblocks that kept us from confidently declaring Wayland's feature-parity with X11, and neatly rounds out the last major holdout towards embedded-window support across all major platforms.
+
+And more:
+- Wayland: Implement the xdg-toplevel-icon-v1 protocol ([GH-107096](https://github.com/godotengine/godot/pull/107096)).
+
+#### Windows
+
+[Skyth](https://github.com/blueskythlikesclouds) was tasked by the Godot Foundation to make Direct3D 12 the default RenderingDevice driver on Windows ([GH-113213](https://github.com/godotengine/godot/pull/113213)). As a result, Windows users can safely sidestep issues with comparatively outdated Vulkan drivers, as Direct3D 12 is the mainstay renderer for modern programs and API kits.
```

Comment:
```suggestion
[Skyth](https://github.com/blueskythlikesclouds) was tasked by the Godot Foundation to bring Direct3D 12 to feature and stability parity with Vulkan, so that we could make it the default RenderingDevice driver on Windows ([GH-113213](https://github.com/godotengine/godot/pull/113213)). As a result, Windows users can safely sidestep issues with comparatively outdated Vulkan drivers, as Direct3D 12 is the mainstay renderer for modern programs and API kits.
```